src/main/java/Demo.java

```diff
@@ -1,7 +1,100 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Demo {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculator1();
+    }
+
+    public static void calculator1() {
```

Comment:
Remember what functions are?
Functions are actions, therefore should be named after verbs, not nouns.
It's not a calculator.
It's probably `run` which runs your demo calculator app?